RELEASE_NOTES.md

```diff
@@ -1,5 +1,19 @@
 # Release Notes
 
+## v1.14.0
+
+* bridge: Add support for a new option called `mode` When set to `unmanaged` only the veth pair and ip addresses are setup. The bridge must exist and no firewall or sysctl setting will be configured in this mode. ([#1090](https://github.com/containers/netavark/issues/1090))
+* bridge: Add support for DHCP when using unmanaged mode. ([#868](https://github.com/containers/netavark/issues/868))
+* bridge: Add support for the `vlan` option. ([#1028](https://github.com/containers/netavark/issues/1028))
+* When using DHCP netavark will now send the container hostname in the DHCP request and use the container id as client id. ([#676](https://github.com/containers/netavark/issues/676))
+* The firewalld driver was improved but it still considered experimental. A new man page `netavark-firewalld(7)` has been added to document some of firewalld interactions.
```

Comment:
Maybe also mention "major outstanding bugs were addressed"

Reply:
if you use the term major, then I think we need to document those no?

Reply:
PTAL again